pom.xml

```diff
@@ -61,7 +62,8 @@
         <maven.compiler.target>8</maven.compiler.target>
         <persistence-api.version>1.0.2</persistence-api.version>
         <resteasy-extension.ver>1.0.0.Final</resteasy-extension.ver>
-        <resteasy.ver>4.5.6.Final</resteasy.ver>
+<!--        <resteasy.ver>4.5.6.Final</resteasy.ver>-->
```

Comment:
Remove commented out line maybe

Reply:
I'll change it to the latest release we have right now.